dns/nextdns-community/src/opnsense/service/templates/OPNsense/Nextdns/nextdns

```diff
@@ -0,0 +1,7 @@
+{% if helpers.exists('OPNsense.nextdns.general.enabled') and OPNsense.nextdns.general.enabled == '1' %}
```

Comment:
This file requires a rc script. 
Just use the one from AdguardHome and replace with your values

Reply:
ok, perhaps the rc script is the missing link why this thing didn't work as I hoped...
(you are making it hard to see AdguardHome [or any other] scripts as you are not publishing it in the repo. I need to install the package and trace through the installed files. Just saying. :-) )